packages/mds-policy-service/service/helpers.ts

```diff
@@ -10,11 +10,17 @@ import type {
 } from '../@types/intents'
 import { BASE_POLICY_DEFAULTS, INTENT_RULE_CONSTANTS } from '../@types/intents'
 
+export const TWENTY_MINUTES = 1000 * 60 * 20
+
 export function translateIntentToPolicy<
   I extends INTENT_TYPE = 'no_parking',
   INTENT_POLICY_FIELDS extends BaseIntentPolicyUserFields = NoParkingIntentPolicyUserFields,
   INTENT_RULE_TYPE extends BaseIntentRuleUserFields = NoParkingIntentRuleUserFields
 >(draft: IntentDraft<I, INTENT_POLICY_FIELDS, INTENT_RULE_TYPE>): PolicyDomainModel {
+  const { start_date } = draft.policy_fields
+  if (!isDefined(start_date) || start_date - now() < TWENTY_MINUTES) {
+    draft.policy_fields.start_date = now() + TWENTY_MINUTES + 1000
```

Comment:
Is the `+ 1000` here a fudge-factor for processing time? If so might be worth making a constant for documentation purposes (e.g. `POLICY_START_DATE_FUDGE_FACTOR = 1000`)